vaadin-text-field-flow-parent/vaadin-text-field-flow/src/test/java/com/vaadin/flow/component/textfield/tests/BigDecimalFieldTest.java

```diff
@@ -52,9 +60,41 @@ public void setValueNull() {
 
     @Override
     @Test
-    public void initialValuePropertyValue() {
-        assertEquals(field.getEmptyValue(),
-                field.getElement().getProperty("value"));
+    public void initialValueIsNotSpecified_valuePropertyHasEmptyString() {
+        BigDecimalField bigDecimalField = new BigDecimalField();
+        Assert.assertNull(bigDecimalField.getValue());
+        Assert.assertEquals("",
+                bigDecimalField.getElement().getProperty("value"));
+    }
+
+    @Override
+    @Test
+    public void initialValueIsNull_valuePropertyHasEmptyString() {
```

Comment:
**note:** BigDecimalField's constructor doesn't have an overload that accepts an initial value.